meshio/medit/_medit.py

```diff
@@ -6,6 +6,8 @@
 <https://www.ljll.math.upmc.fr/frey/publications/RT-0253.pdf>
 <https://www.math.u-bordeaux.fr/~dobrzyns/logiciels/RT-422/node58.html>
 for something like a specification.
```

Comment:
Are the old references rendered obsolete by libMeshb?  Should they be deleted?

Reply:
Yes the old ones are just a subset.  So I will be keeping only the latest